aea/test_tools/test_cases.py

```diff
@@ -273,7 +278,13 @@ def is_allowed_diff_in_agent_config(
                 if content2[key] == value:
                     content1.pop(key)
                     content2.pop(key)
-            allowed_diff_keys = ["aea_version", "author", "description", "version"]
+            allowed_diff_keys = [
+                "aea_version",
+                "author",
+                "description",
+                "version",
+                "registry_path",
```

Comment:
Why has a key been added?

Reply:
cause it differs on windows and linux, cause slash in path

Reply:
then we should use platform type and have if/else. Just a bit stricter.